source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0010
+
+<b>Functional Description</b>
+
+IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '12'<br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Codelist number against which validation failed (ie CLxxx)<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {13, 15} <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Condition/Technical Rule number against which validation failed (ie Cxxxx or Txxxx),<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '14' <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Rules/Technical Rule number against which validation failed (ie Rxxxx or Txxxx)<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is EQUAL to '50' <br/>THEN &lt;FUNCTIONAL ERROR.Error reason&gt; shall point to the Transitional Constraint number against which validation failed (ie Exxxx or Bxxxx),<br/>ELSE IF &lt;FUNCTIONAL ERROR.Error code&gt; is in SET {51, 52} <br/>THEN the &lt;FUNCTIONAL ERROR.Error reason&gt; shall be:<br/>-  'ieCAvB' if exception is thrown by ieCA<br/>-  'NCAvB' if exception is thrown by NTA/NECA,<br/>ELSE the &lt;FUNCTIONAL ERROR.Error reason&gt; shall have the value 'N/A'
+
+<b>Technical Description</b>
+
+N/A
+
+##G0011
+
+<b>Functional Description</b>
+
+It contains the text of the error returned by the XML parser or XML validator.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0012
+
+<b>Functional Description</b>
+
+It should include the XPath location of the error. If the XPath string is to be truncated (i.e. if the length of the string is greater than 512 characters long), then the data item should not be used.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0013
+
+<b>Functional Description</b>
+
+It should be used when the error is an XML schema error concerning invalid values. The reasons for considering an attribute value invalid might be the format and/or a value for a technical code list. For such cases, the data item should contain the value of the invalid value in order to indicate which value was perceived invalid.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0014
+
+<b>Functional Description</b>
+
+Eastern longitude and Northern latitude will use the optional '+' sign.<br/>Western longitude and Southern latitude will use the '-' sign.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0015
+
+<b>Functional Description</b>
+
+In case of Incident information received via one or multiple CD180C, all the information received shall be included in the D.G. &lt;CONSIGNMENT-INCIDENT&gt;.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0016
+
+<b>Functional Description</b>
+
+&lt;CONSIGNMENT-INCIDENT-TRANSPORT EQUIPMENT.Container identification number&gt; is filled in case of incident with the new Container identification number amending initial declaration or with the existing Container identification number if DI Number of seals is greater than 0.
+
```

Comment:
Grammar/style issues -
- `in case of incident` does not read well. Can we reword?
- `amending initial declaration` does not read well. Need to insert `an` or `the `before initial.